src/ttsim/interface_dag_elements/targets.py

```diff
@@ -15,13 +15,19 @@
         OrderedQNames,
     )
 
+NamespaceSelector = Literal["all"] | str  # noqa: PYI051
+
 
 @interface_input()
 def tree() -> NestedTargetDict:
-    """All targets as a tree. If requesting `df_with_mapper` as the targets, the leaves must be the desired column names."""  # noqa: E501
+    """Targets as a tree.
+
+    If requesting `df_with_mapper` as the targets, the leaves must be the desired
+    column names.
+    """
 
 
 @interface_function()
 def qname(tree: NestedTargetDict) -> OrderedQNames:
-    """All targets in their qualified name-representation."""
-    return dt.qual_names(tree)
+    """Targets in their qualified name-representation."""
+    return dt.qnames(tree)
```

Comment:
```suggestion
    return dt.flatten_to_qnames(tree)
```
We'll should not get rid of the leaves here or make that clear in the name, I think.